smart-contracts/scripts/helpers/ofacParser.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * This will parse the OFAC list, extracting EVM addresses
+ * It will also convert addresses to their checksum version
+ * And remove any duplicate addresses found in OFAC's list
+ */
+
+const Web3 = require("web3");
+const web3 = new Web3();
+const axios = require("axios");
+const { print, cacheBuster, removeDuplicates } = require("./utils");
+
+const OFAC_URL = "https://www.treasury.gov/ofac/downloads/sdnlist.txt";
+
+// Make this true to test a diff
+const FORCE_DIFF_FOR_TESTING = false;
+
+async function getList() {
+  print("yellow", "Fetching and parsing OFAC blocklist. Please wait...");
+
+  const finalUrl = cacheBuster(OFAC_URL);
+  const response = await axios.get(finalUrl).catch((e) => {
+    throw e;
+  });
+
+  const addresses = extractAddresses(response.data);
+
+  if (FORCE_DIFF_FOR_TESTING) {
```

Comment:
Should we keep this?

Reply:
I think you're right; it's already tested and I don't see why we should keep this. I'm removing it.